sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -434,7 +434,6 @@ abstract class ScrollableListPainter extends Painter {
   double _scrollOffset = 0.0;
   void set scrollOffset (double value) {
     assert(value != null);
-    assert(value >= 0.0 && value <= 1.0);
```

Comment:
OMG I've been running with checking turned off.

sky_tool --help doesn't help
sky_tool start --help delivers the goods

Not sure when it became necessary to specify --checked explicitly. I've checked old examples of mine, they seem to be OK.

https://github.com/flutter/engine/pull/1644